### PR TITLE
[update] SQL function API Reference

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

[Short summary of why you created this PR]

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links
Fixes https://github.com/timescale/docs/issues/837
replaces https://github.com/timescale/docs/pull/997

### CODE REVIEW

File: promscale/sql-api.md [hunk 1]
```diff
@@ -17,23 +17,37 @@ WHERE n.nspname OPERATOR(pg_catalog.~) '^(prom)$' COLLATE pg_catalog.default
...
- set_default_retention_period  | retention_period interval                                | boolean          | set_default_retention_period set the retention period for any metrics (existing and new) without an explicit override.
- set_metric_chunk_interval     | metric_name text, chunk_interval interval                | boolean          | set_metric_chunk_interval set a chunk interval for a specific metric (this overrides the default).
- set_metric_retention_period   | metric_name text, new_retention_period interval          | boolean          | set_metric_retention_period set a retention period for a specific metric (this overrides the default).
- val                           | label_id integer                                         | text             | val returns the label value from a label id.
- unregister_metric_view        | schema_name text, view_name text, if_not_exists boolean  | boolean          | Unregister metric view with Promscale. Schema name and view name should be set to the metric view already registered in Promscale. 
\ No newline at end of file
+ |Name|Arguments|Type|Description|
+|-|:-:|:-:|:-|
```
Comment (Loquacity) [inline promscale/sql-api.md:21]: The colons aren't being honoured here. I think maybe Gatsby ignores them?

File: promscale/sql-api.md [hunk 2]
```diff
@@ -17,23 +17,37 @@ WHERE n.nspname OPERATOR(pg_catalog.~) '^(prom)$' COLLATE pg_catalog.default
...
- set_default_retention_period  | retention_period interval                                | boolean          | set_default_retention_period set the retention period for any metrics (existing and new) without an explicit override.
- set_metric_chunk_interval     | metric_name text, chunk_interval interval                | boolean          | set_metric_chunk_interval set a chunk interval for a specific metric (this overrides the default).
- set_metric_retention_period   | metric_name text, new_retention_period interval          | boolean          | set_metric_retention_period set a retention period for a specific metric (this overrides the default).
- val                           | label_id integer                                         | text             | val returns the label value from a label id.
- unregister_metric_view        | schema_name text, view_name text, if_not_exists boolean  | boolean          | Unregister metric view with Promscale. Schema name and view name should be set to the metric view already registered in Promscale. 
\ No newline at end of file
+ |Name|Arguments|Type|Description|
```
Comment (Loquacity) [inline promscale/sql-api.md:20]:
```suggestion
|Name|Arguments|Type|Description|
```

Comment (Loquacity) [reply promscale/sql-api.md:20]: Left alignment is out throughout the table.

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Comment by Loquacity** [comment]
@Rajakavitha1 This hasn't resolved the problem of the right column scrolling off the side. We have a very narrow column for main content, and to scroll right in a table, readers need to go all the way to the bottom to find the grab bar. You probably need to re-think this table.

File: promscale/sql-api.md (hunk 1 above)
Comment (Rajakavitha1) [reply promscale/sql-api.md:21]: Yes!!! Looks like Gatsby ignores them. I will redo the table may be.

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 This hasn't resolved the problem of the right column scrolling off the side. We have a very narrow column for main content, and to scroll right in a table, readers need to go all the way to the bottom to find the grab bar. You probably need to re-think this table.

Hi @Loquacity !!! I have now created new smaller tables based on functions. Please let me know if this is okay?

**Comment by Loquacity** [comment]
It's certainly better than it was, let's get it merged.

**Review by Loquacity — APPROVED** [review]

**Review by VineethReddy02 — APPROVED** [review]
LGTM!

**Pull request merged**